### PR TITLE
feat(TDP-1832): newsletter puff restyling

### PR DESCRIPTION
**Pull request by SunshineEleda**
#### Changes
- restyling
- refactoring

https://nidigitalsolutions.jira.com/browse/TDP-1832

Mobile
Screenshot 2022-10-07 at 12 27 06

Screenshot 2022-10-07 at 12 28 07

Screenshot 2022-10-07 at 12 27 48

Desktop landscape
Screenshot 2022-10-07 at 12 29 13

Screenshot 2022-10-07 at 12 28 55

Screenshot 2022-10-07 at 12 29 26

### CODE REVIEW

File: packages/ts-components/src/components/newsletter-puff/__tests__/InlineNewsletterPuff.test.tsx [hunk 1]
```diff
@@ -69,13 +69,10 @@ const renderComponent = (
         <InlineNewsletterPuff
           {...{
             code: 'TNL-119',
-
-            label: 'STRAIGHT IN YOUR INBOX',
+            sectionColour: 'news',
```
Comment (bibicollins1) [inline packages/ts-components/src/components/newsletter-puff/__tests__/InlineNewsletterPuff.test.tsx:72]: Just one thing, should this be section: 'news'?

**Review by bibicollins1 — APPROVED** [review]

File: packages/ts-components/src/components/newsletter-puff/newsletter-puff-button/styles.ts [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  fontFamily: fonts.supporting,
+  height: 48,
+  justifyContent: 'center',
+  letterSpacing: 0.2,
+  width: '100%'
+};
+
+export const LinkButton = styled.button`
+  color: ${colours.functional.action};
+
```
Comment (othetgyi) [inline packages/ts-components/src/components/newsletter-puff/newsletter-puff-button/styles.ts:22]: Very minor comment: do we need the extra line here?

File: packages/ts-components/src/components/newsletter-puff/loading-overlay/styles.ts [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+export const Bubble = styled.div`
+  animation: expand 0.75s ease-in-out infinite;
+  border-radius: 20px;
+  display: inline-block;
+  transform-origin: center center;
+  margin: 0 7px;
+  width: 16px;
+  height: 16px;
+
```
Comment (othetgyi) [inline packages/ts-components/src/components/newsletter-puff/loading-overlay/styles.ts:27]: Is this space between height and background intentional? If not, maybe we can remove it?

File: packages/ts-components/src/components/newsletter-puff/loading-overlay/styles.ts [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+    100% {
+      background: #6b6b6b;
+    }
+  }
+`;
+
+export const Loader = styled(View)`
+  display: flex;
+  flex-direction: row;
+
```
Comment (othetgyi) [inline packages/ts-components/src/components/newsletter-puff/loading-overlay/styles.ts:51]: Same line space comment as above.

File: packages/ts-components/src/components/newsletter-puff/loading-overlay/styles.ts [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+  border-radius: 20px;
+  display: inline-block;
+  transform-origin: center center;
+  margin: 0 7px;
+  width: 16px;
+  height: 16px;
+
+  background: #c4c4c4;
+
+  :nth-child(1) {
```
Comment (othetgyi) [inline packages/ts-components/src/components/newsletter-puff/loading-overlay/styles.ts:30]: Should this pseudo-class have some styling, or if not, can it be removed?

**Review by othetgyi — COMMENTED** [review]
Nice work in refactoring the newsletter puff, Adele! I had just a few very minor comments about spacing but other than that it looked great! :)

**Pull request merged**